gotrue/_async/gotrue_client.py

```diff
@@ -483,10 +484,15 @@ async def sign_out(self) -> None:
         There is no way to revoke a user's access token jwt until it expires.
         It is recommended to set a shorter expiry on the jwt for this reason.
         """
-        session = await self.get_session()
-        access_token = session.access_token if session else None
-        if access_token:
-            await self.admin.sign_out(access_token)
+        try:
+            session = await self.get_session()
+            access_token = session.access_token if session else None
+            if access_token:
+                await self.admin.sign_out(access_token)
+        except AuthApiError:
+            pass
```

Comment:
Not necessary, but I generally prefer using [`contextlib.suppress`](https://docs.python.org/3/library/contextlib.html#contextlib.suppress) instead of an `except ...: pass` - the intent becomes more obvious imo

Reply:
I've learnt something new today and yeah I agree, let me change that.